src/index.ts

```diff
@@ -64,6 +65,20 @@ export function createStore<PrivateKeyData, PublicKeyData = {}> (
   const { iterations = 10000 } = options
   let keysData = initialKeys
 
+  function _saveKey (keyID: string, password: string, privateData: PrivateKeyData, publicData: PublicKeyData | {} = {}): void {
```

Comment:
Let's drop the leading `_`. Will be called the same as the store method then, but that's fine, since it also does the very same thing :)